### PR TITLE
Added citations for other packages and softwares 

### PR DESCRIPTION
**Pull request by cying111**
# nf-core/nanoseq pull request

Many thanks for contributing to nf-core/nanoseq!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)

- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/nanoseq branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/nanoseq)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/nanoseq/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: CITATIONS.md [hunk 1]
```diff
@@ -18,9 +18,25 @@
 * [featureCounts](https://www.ncbi.nlm.nih.gov/pubmed/24227677/)
   > Liao Y, Smyth GK, Shi W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Bioinformatics. 2014 Apr 1;30(7):923-30. doi: 10.1093/bioinformatics/btt656. Epub 2013 Nov 13. PubMed PMID: 24227677.
 
+* [GraphMap](https://pubmed.ncbi.nlm.nih.gov/27079541/)
+  > Sović I, Šikić M, Wilm A, Fenlon SN, Chen S, Nagarajan N. Fast and sensitive mapping of nanopore sequencing reads with GraphMap. Nat Commun. 2016 Apr 15;7:11307. doi: 10.1038/ncomms11307. PMID: 27079541; PMCID: PMC4835549.
+ 
+* [Guppy](https://hpc.nih.gov/apps/guppy.html)
+  > R. R. Wick, L. M. Judd, and K. E. Holt. Performance of neural network basecalling tools for Oxford Nanopore sequencing. Genome Biology 2019, 20:129. doi: 10.1186/s13059-019-1727-y PubMed PMID: 31234903; PubMed Central PMCID: PMC6591954.
```
Comment (drpatelh) [inline CITATIONS.md:25]:
```suggestion
  > Wick RR, Judd LM, and Holt KE. Performance of neural network basecalling tools for Oxford Nanopore sequencing. Genome Biology 2019, 20:129. doi: 10.1186/s13059-019-1727-y. PubMed PMID: 31234903; PubMed Central PMCID: PMC6591954.

```

File: CITATIONS.md [hunk 2]
```diff
@@ -18,9 +18,25 @@
...
+* [Minimap2](https://pubmed.ncbi.nlm.nih.gov/29750242/)
+  > Li H. Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics. 2018 Sep 15;34(18):3094-3100. doi: 10.1093/bioinformatics/bty191. PMID: 29750242; PMCID: PMC6137996.
 * [MultiQC](https://www.ncbi.nlm.nih.gov/pubmed/27312411/)
   > Ewels P, Magnusson M, Lundin S, Käller M. MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics. 2016 Oct 1;32(19):3047-8. doi: 10.1093/bioinformatics/btw354. Epub 2016 Jun 16. PubMed PMID: 27312411; PubMed Central PMCID: PMC5039924.
 
+* [NanoPlot](https://academic.oup.com/bioinformatics/article/34/15/2666/4934939)
+  > De Coster W, D'Hert S, Schultz DT, Cruts M, Van Broeckhoven C. NanoPack: visualizing and processing long-read sequencing data. Bioinformatics. 2018 Aug 1;34(15):2666-2669. doi: 10.1093/bioinformatics/bty149. PMID: 29547981; PMCID: PMC6061794.
+
+* [pycoQC](https://doi.org/10.21105/joss.01236)
+  > Leger et al., (2019). pycoQC, interactive quality control for Oxford Nanopore Sequencing. Journal of Open Source Software, 4(34), 1236, https://doi.org/10.21105/joss.01236
```
Comment (drpatelh) [inline CITATIONS.md:36]: Full author list required here.

File: CITATIONS.md [hunk 3]
```diff
@@ -18,9 +18,25 @@
...
+  > R. R. Wick, L. M. Judd, and K. E. Holt. Performance of neural network basecalling tools for Oxford Nanopore sequencing. Genome Biology 2019, 20:129. doi: 10.1186/s13059-019-1727-y PubMed PMID: 31234903; PubMed Central PMCID: PMC6591954.
+
+* [Minimap2](https://pubmed.ncbi.nlm.nih.gov/29750242/)
+  > Li H. Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics. 2018 Sep 15;34(18):3094-3100. doi: 10.1093/bioinformatics/bty191. PMID: 29750242; PMCID: PMC6137996.
 * [MultiQC](https://www.ncbi.nlm.nih.gov/pubmed/27312411/)
   > Ewels P, Magnusson M, Lundin S, Käller M. MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics. 2016 Oct 1;32(19):3047-8. doi: 10.1093/bioinformatics/btw354. Epub 2016 Jun 16. PubMed PMID: 27312411; PubMed Central PMCID: PMC5039924.
 
+* [NanoPlot](https://academic.oup.com/bioinformatics/article/34/15/2666/4934939)
+  > De Coster W, D'Hert S, Schultz DT, Cruts M, Van Broeckhoven C. NanoPack: visualizing and processing long-read sequencing data. Bioinformatics. 2018 Aug 1;34(15):2666-2669. doi: 10.1093/bioinformatics/bty149. PMID: 29547981; PMCID: PMC6061794.
```
Comment (drpatelh) [inline CITATIONS.md:33]: Thats the wrong pubmed ID? 
https://pubmed.ncbi.nlm.nih.gov/29547981/

**Review by drpatelh — CHANGES_REQUESTED** [review]
Can you just double-check all of these to be consistent based on the comments @cying111

File: CITATIONS.md (hunk 3 above)
Comment (cying111) [reply CITATIONS.md]: Just changed, pls check.

File: CITATIONS.md (hunk 2 above)
Comment (cying111) [reply CITATIONS.md]: Completed the author list for pycoQC

**Comment by drpatelh** [comment]
Thanks @cying111 !

**Review by drpatelh — APPROVED** [review]

**Pull request merged**